test/utils/logger.test.js

```diff
@@ -0,0 +1,81 @@
+const mockDate = require('mockdate');
+
+const config = require('../../lib/config');
+const { log } = require('../../lib/utils/logger');
+
+// test setup
+const timestamp = Date.now();
+
+const originalConsole = console;
+const mockConsole = { log: jest.fn() };
+// freeze time
+mockDate.set(timestamp);
+
+describe('log()', () => {
+  beforeAll(() => {
+    // eslint-disable-next-line no-global-assign
+    console = mockConsole;
+    config.set('loggerMode', 'normal');
+  });
+  afterAll(() => {
+    // eslint-disable-next-line no-global-assign
+    console = originalConsole;
+    mockDate.reset();
+  });
+
+  const expectedDate = new Date(timestamp);
+
+  test('logs are printed in the correct format', () => {
+    log('testLog');
+
+    expect(mockConsole.log).toHaveBeenCalledWith(
+      expect.stringContaining(`RewardOps SDK INFO[${expectedDate.toISOString()}] testLog`)
+    );
+  });
+
+  test('options.level is present in the log', () => {
+    log('errorLog', { level: 'error' });
+
+    expect(mockConsole.log).toHaveBeenCalledWith(
+      expect.stringContaining(`RewardOps SDK ERROR[${expectedDate.toISOString()}] errorLog`)
+    );
+  });
+
+  test('instances of Error are parsed into the log', () => {
+    const error = new Error();
+    error.message = 'errorMessage';
+    error.stack = 'errorStack';
+
+    log(error);
+
+    expect(mockConsole.log).toHaveBeenCalledWith(
+      expect.stringContaining(`RewardOps SDK ERROR[${expectedDate.toISOString()}] ${error.message}\n${error.stack}`)
```

Comment:
Optional: For less brittleness, I'd consider breaking up the `stringContaining` assertions, just in case we tweak the message format. Hmm. Maybe, maybe not…

Reply:
Good point, we can change it if you want. 

I just thought it would be easier to read the test and see what the log looked like.

Reply:
Yeah… We can leave it for now